### ISSUE TITLE
Support for other BSDs

### ISSUE THREAD

**Issue by ghost** [IsaacShelton/Adept3x#19]
Once the support for FreeBSD is ready, support for other BSDs is pretty much just copy/paste.

**Comment by IsaacShelton** [comment]
Yes, as mentioned in #18, Linux and the BSDs use the same binary format and C calling convention, so it should be pretty easy, we just need copies of the C standard library from each platform
